composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "mindplay/annotations",
+    "name": "Naerymdan/annotations",
```

Comment:
Ah that was simply to support my local composer pull, it's not supposed to be part of the pull request.